pkgs/development/python-modules/lark-parser/default.nix

```diff
@@ -1,28 +1,34 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
+  # Test inputs
+, pytestCheckHook
+, regex
 }:
 
 buildPythonPackage rec {
   pname = "lark-parser";
-  version = "0.8.8";
+  version = "0.11.2";
 
   src = fetchFromGitHub {
     owner = "lark-parser";
     repo = "lark";
     rev = version;
-    sha256 = "1q2dvkkfx9dvag5v5ps0ki4avh7i003gn9sj30jy1rsv1bg4y2mb";
+    sha256 = "1v1piaxpz4780km2z5i6sr9ygi9wpn09yyh999b3f4y0dcz20pbd";
   };
 
-  # tests of Nearley support require js2py
-  preCheck = ''
-    rm -r tests/test_nearley
-  '';
+  checkInputs = [ pytestCheckHook regex ];
```

Comment:
I think it makes sense to put `regex` in `propagatedBuilldInputs` since it's in `extra_requires`.

Reply:
Good point, thanks.